shared/params/network_config.go

```diff
@@ -42,7 +42,7 @@ func BeaconNetworkConfig() *NetworkConfig {
 // OverrideBeaconNetworkConfig will override the network
 // config with the added argument.
 func OverrideBeaconNetworkConfig(cfg *NetworkConfig) {
-	networkConfig = cfg.Copy()
+	networkConfig = cfg
```

Comment:
In your description it says it's only for tests but this is not used only for tests

Reply:
You're right... Now I am not so sure about the safety of this change. On the other hand this will align the network config's setup with the beacon config, so it doesn't look risky.

Reply:
At the minimal we should test pyrmont or prater still works. Better see it breaking earlier than later

Reply:
Lets leave it as it is, is there any reason a `Copy` needs to be removed ?